reducers/.idea/test_feature_selectors.py

```diff
@@ -0,0 +1,13 @@
+import sys
+import os
+sys.path.insert(0, os.path.abspath('..'))
+
+from unittest import TestCase
+from modules.toolbox import framework_tools as ft
+from modules.toolbox import ml_runners as mr
+from reducers import scikit_feature_selectors
+import setup
+
+class TestFeatureSelectors(TestCast):
+
+    def test_VarianceThreshold_selector(self):
```

Comment:
@Zak7062 this is nit picky, so I'm sorry. But function names, according to Google standards, should be like this "function_name"